package.json

```diff
@@ -68,6 +68,7 @@
     "docker:nuke": "docker rmi $(docker images -f=\"reference=cuttle*\" -q --no-trunc) || true; npm run docker:clean",
     "docker:start": "docker compose -f ./docker/docker-compose.yml up",
     "docker:stop": "docker compose -f ./docker/docker-compose.yml down",
+    "docker:e2e": "docker compose -f ./docker/docker-compose.yml -f ./docker/docker-compose-e2e.yml up --exit-code-from e2e",
```

Comment:
@itsalaidbacklife you can run this to try it out but the tests all fail:
```
e2e       | http://localhost:1337/test/wipeDatabase
e2e       | 
e2e       | We attempted to make an http request to this URL but the request failed without a response.
e2e       | 
e2e       | We received this error at the network level:
e2e       | 
e2e       |   > Error: connect ECONNREFUSED 127.0.0.1:1337
e2e       | 
e2e       | -----------------------------------------------------------
e2e       | 
e2e       | The request we sent was:
e2e       | 
e2e       | Method: GET
e2e       | URL: http://localhost:1337/test/wipeDatabase
e2e       | 
e2e       | -----------------------------------------------------------
e2e       | 
e2e       | Common situations why this would fail:
e2e       |   - you don't have internet access
e2e       |   - you forgot to run / boot your web server
e2e       |   - your web server isn't accessible
e2e       |   - you have weird network configuration settings on your computer
e2e       | 
e2e       | https://on.cypress.io/request
e2e       | 
e2e       | Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Signing Up`
e2e       |       at <unknown> (http://client:8080/__cypress/runner/cypress_runner.js:149355:78)
e2e       |       at tryCatcher (http://client:8080/__cypress/runner/cypress_runner.js:18744:23)
e2e       |       at <unknown> (http://client:8080/__cypress/runner/cypress_runner.js:13883:37)
e2e       |       at tryCatcher (http://client:8080/__cypress/runner/cypress_runner.js:18744:23)
e2e       |       at Promise._settlePromiseFromHandler (http://client:8080/__cypress/runner/cypress_runner.js:16679:31)
e2e       |       at Promise._settlePromise (http://client:8080/__cypress/runner/cypress_runner.js:16736:18)
e2e       |       at Promise._settlePromise0 (http://client:8080/__cypress/runner/cypress_runner.js:16781:10)
e2e       |       at Promise._settlePromises (http://client:8080/__cypress/runner/cypress_runner.js:16857:18)
e2e       |       at _drainQueueStep (http://client:8080/__cypress/runner/cypress_runner.js:13451:12)
e2e       |       at _drainQueue (http://client:8080/__cypress/runner/cypress_runner.js:13444:9)
e2e       |       at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://client:8080/__cypress/runner/cypress_runner.js:13460:5)
e2e       |       at Async.drainQueues (http://client:8080/__cypress/runner/cypress_runner.js:13330:14)
e2e       |   From Your Spec Code:
e2e       |       at Context.eval (webpack:///./tests/e2e/support/commands.js:15:5)
e2e       |   
e2e       |   From Node.js Internals:
e2e       |     RequestError: Error: connect ECONNREFUSED 127.0.0.1:1337
e2e       |         at new RequestError (/root/.cache/Cypress/12.17.3/Cypress/resources/app/node_modules/request-promise-core/lib/errors.js:14:15)
e2e       |         at Request.plumbing.callback (/root/.cache/Cypress/12.17.3/Cypress/resources/app/node_modules/request-promise-core/lib/plumbing.js:87:29)
e2e       |         at Request.RP$callback [as _callback] (/root/.cache/Cypress/12.17.3/Cypress/resources/app/node_modules/request-promise-core/lib/plumbing.js:46:31)
e2e       |         at self.callback (/root/.cache/Cypress/12.17.3/Cypress/resources/app/node_modules/@cypress/request/request.js:183:22)
e2e       |         at Request.emit (node:events:527:28)
e2e       |         at Request.emit (node:domain:475:12)
e2e       |         at Request.onRequestError (/root/.cache/Cypress/12.17.3/Cypress/resources/app/node_modules/@cypress/request/request.js:869:8)
e2e       |         at ClientRequest.emit (node:events:539:35)
e2e       |         at ClientRequest.emit (node:domain:475:12)
e2e       |         at Socket.socketErrorListener (node:_http_client:454:9)
e2e       |         at Socket.emit (node:events:527:28)
e2e       |         at Socket.emit (node:domain:475:12)
e2e       |         at emitErrorNT (node:internal/streams/destroy:157:8)
e2e       |         at emitErrorCloseNT (node:internal/streams/destroy:122:3)
e2e       |         at processTicksAndRejections (node:internal/process/task_queues:83:21)
```